code/game/machinery/computer/cargo.dm

```diff
@@ -18,6 +18,9 @@ For vending packs, see vending_packs.dm*/
 	else if(issilicon(user))
```

Comment:
Did you test it ?
`issilicon(user)` should pass if the user is an AI - the check you added will never trigger.
Change the order of the checks.